ios/FluentUI.Demo/FluentUI.Demo.xcodeproj/project.pbxproj

```diff
@@ -801,7 +801,7 @@
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				CODE_SIGN_STYLE = Manual;
-				DEVELOPMENT_TEAM = 9KBH5RKYEW;
+				DEVELOPMENT_TEAM = UBF8T346G9;
```

Comment:
These should not be changed. If they are, then it means we've been distributing non-dogfood builds with the dogfood certificate. @rdeassis can you confirm?

Reply:
We're not going to be used the development AppID (AppStore) anymore.
ESRP requires the app to be submitted as a debug build with the same AppID as the dogfood one. We're only using the -df AppID now since the demo app is only distributed internally.

Reply:
As discussed offline, we'll restore Debug and Release schemes to the original bundle IDs and dev teams, while updating the Dogfood scheme to match our internal ESRP needs.